Frends.Regex.MatchResult/Frends.Regex.MatchResult/Frends.Regex.MatchResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Frends.Regex.MatchResult;
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using Frends.Regex.MatchResult.Definitions;
+
+/// <summary>
+/// Regex MatchResult Task.
+/// </summary>
+public static class Regex
+{
+    /// <summary>
+    /// Frends Task will return full info for match result.
+    /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.Regex.MatchResult).
+    /// </summary>
+    /// <param name="input">What to repeat.</param>
+    /// <param name="cancellationToken">Cancellation token given by Frends.</param>
+    /// <returns>MatchCollection matches</returns>
+    public static Result MatchResult([PropertyTab] Input input, CancellationToken cancellationToken)
```

Comment:
Lets make this called just Match. MatchResult is somewhat redundant :) Sorry for specifying this a bit too late.